ui/components/ui/new-network-info/index.scss

```diff
@@ -0,0 +1,52 @@
+.new-network-info {
+  &__wrapper {
+    box-shadow: 0 4px 20px rgba(0, 0, 0, 0.214);
+    border-radius: 8px;
+
+    & .popover-footer {
```

Comment:
I don't think all of these `& ` are necessary.

Reply:
> I really like this implementation! Well done!
> 
> One thing I noticed -- the design mockups show an ordered list "1, 2, 3", but in this PR I'm seeing bullets. Was there a design change or should we update the list type?

There was a design change. You can see it within the conversation on Slack. (https://consensys.slack.com/archives/C02GH833686/p1638972279135800?thread_ts=1638965448.130400&cid=C02GH833686) 

Fixe in: 3212de0acbc7e48bb460c8bc2bd71fe99a2f70c5